manifests/b/BornToBeRoot/NETworkManager/2024.6.15.0/BornToBeRoot.NETworkManager.installer.yaml

```diff
@@ -9,6 +9,9 @@ Installers:
   InstallerUrl: https://github.com/BornToBeRoot/NETworkManager/releases/download/2024.6.15.0/NETworkManager_2024.6.15.0_Setup.msi
   InstallerSha256: cb78e596ef0247808522b6f5883cd154bf0a8630496e2545aaf99e1b324e3d9b
 ProductCode: '{e343d8ab-ed71-41d1-9784-6f6860bda078}'
+AppsAndFeaturesEntries:
+- DisplayName: NETworkManager 2024.6.15.0
+  DisplayVersion: 24.6.15
```

Comment:
Wait

Reply:
I read the PackageVersion wrong - didn’t see that the PackageVersion was `2024.` and not just `24.`

Reply:
So now we run into the problem where we can’t change the PackageVersion because it will always be less than the existing package versions. 

If we add DisplayVersion here, all other versions will simply show as `< 2024.6.15.0`. Which technically isn’t wrong. . . 

Its just that all versions moving forward would need the DisplayVersion regardless of if the PackageVersion matched or not

(sorry for all the confusion, I'm a bit tired)

Reply:
Can be merged